integration/kubernetes/runtimeclass_workloads/pod-footloose.yaml

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+apiVersion: v1
+kind: Pod
+metadata:
+  name: footfed
+spec:
+  runtimeClassName: kata
+  volumes:
+  - name: runv
+    emptyDir:
+      medium: "Memory"
+  - name: runlockv
+    emptyDir:
+      medium: "Memory"
+  - name: tmpv
+    emptyDir:
+      medium: "Memory"
+  - name: fakecgroup
+    hostPath:
+      path: /sys/fs/cgroup
+  - name: ssh-dir
+    emptyDir:
+      medium: "Memory"
+  - name: ssh-pub-key
+    configMap:
+      name: ssh-pub-key
+      defaultMode: 384
+  containers:
+  - name: vmcontainer
+    image: quay.io/footloose/fedora29:latest
```

Comment:
This version number is rather "buried" here. Can you specify `fedora:latest` instead?

Reply:
@jodh-intel no, if I used `fedora:latest` the pod status is
```
NAME      READY   STATUS             RESTARTS   AGE
footfed   0/1     ImagePullBackOff   0          48s
```

Reply:
I still don't like the idea of hardcoding a distro version which will one day go EOL. We're just storing up problems for ourselves in the future. *Maybe* the f29 image will continue to exist, or maybe it will be deleted and then this test will fail everywhere.

Is there a particular reason for choosing fedora? This page lists the available images:

- https://quay.io/organization/footloose

... and I note they have `clearlinux`. By using that, you can side-step the problem by avoiding hardcoding a version number:

- https://quay.io/repository/footloose/clearlinux